src/execution/riscv64/simulator-riscv64.h

```diff
@@ -499,8 +500,10 @@ class Simulator : public SimulatorBase {
   }
   inline int16_t shamt() const { return (imm12() & 0x3F); }
   inline int16_t shamt32() const { return (imm12() & 0x1F); }
+  inline int16_t rvc_shamt() const { return instr_.RvcShamt(); }
   inline int32_t s_imm12() const { return instr_.StoreOffset(); }
   inline int32_t u_imm() const { return instr_.Imm20UValue() << 12; }
+  inline int32_t rvc_u_imm() const { return instr_.RvcImm6UValue() << 12; }
```

Comment:
Change `rvc_u_imm` name to reflect the number of bits in the immediate value. It looks like it should be  `rvc_u_imm20`? 

Similarly please update `u_imm` to be `u_imm20`